src/Host/Client/Impl/BrokerServices/RemoteUriWebService.cs

```diff
@@ -13,13 +13,22 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.WebSockets.Protocol;
 using Microsoft.Common.Core;
+using Microsoft.Common.Core.Logging;
+using Microsoft.Common.Core.Services;
 using Microsoft.R.Host.Protocol;
 using static System.FormattableString;
 
 namespace Microsoft.R.Host.Client.BrokerServices {
     public class RemoteUriWebService : IRemoteUriWebService {
-        public RemoteUriWebService(string baseUri) {
+        private ICoreServices _services;
+        private IConsole _console;
+
+        private IActionLog Log => _services?.Log;
```

Comment:
I'd rather use a field and assign it in ctor.
Also, do we have a case when log is null?